docs/knowledgebase/advanced/ss58-address-format.md

```diff
@@ -13,3 +13,95 @@ extensible for this reason.
 The living specification for the SS-58 address format can be found on the Substrate GitHub wiki:
 
 https://github.com/paritytech/substrate/wiki/External-Address-Format-(SS58)
+
+## Validating addresses
+
+There are several ways to verify that a value is a valid SS58 address.
+
+### Subkey
+
+You can use the [Subkey](https://substrate.dev/docs/en/knowledgebase/integrate/subkey) inspect
+subcommand which will accept the seed phrase, an address as hex or an SS58 address as the input
+URI. If the input is a valid address, it will return a list containing the corresponding Public
+key (hex), Account ID, and SS58 values.
```

Comment:
```suggestion
URI. If the input is a valid address, it will return a list containing the corresponding public
key (hex), account ID, and SS58 values.
```